src/java/org/jivesoftware/openfire/plugin/rest/service/MUCRoomService.java

```diff
@@ -240,10 +240,12 @@ public Response inviteUserOrGroupToMUCRoom(
             @RequestBody(description = "The invitation message to send and whom to send it to.", required = true) MUCInvitationEntity mucInvitationEntity)
         throws ServiceException
     {
-        if (!mucInvitationEntity.getJidsToInvite().contains(jid)) {
-            mucInvitationEntity.getJidsToInvite().add(jid);
+        final MUCInvitationsEntity multiple = new MUCInvitationsEntity();
+        multiple.setReason(mucInvitationEntity.getReason());
+        if (!multiple.getJidsToInvite().contains(jid)) {
```

Comment:
This wouldn't have any JIDs at all at this point, right?

Reply:
It shouldn't, no - but this won't remove them if for a reason that I can't see they'd be there.